control/control.openSUSE.xml

```diff
@@ -301,6 +300,13 @@ textdomain="control"
                 </window_manager>
 
                 <window_manager>
+                    <sysconfig_wm>startkde</sysconfig_wm>
```

Comment:
this looks strange to have two time here `startkde` I think it should be only once there. So remove old one or choose new sysconfig_wm for plasma.